lib/laforge/activerecord.rb

```diff
@@ -87,6 +89,8 @@ def record_data_entries(attributes_hash, source, **data_entry_options)
     # Record a single piece of information from a source.
     # Optionally pass a custom priority for that attribute and source at the same time.
     def record_data_entry(attribute_name, value, source, priority: nil)
+      raise InvalidAttributeName, "Cannot set #{attribute_name} on #{self.class.name}" unless respond_to?("#{attribute_name}")
```

Comment:
Shouldn't we be checking the setter here, not the getter? Also, if we don't need to, then there's no need to interpolate the `attribute_name`.